.pre-commit-config.yaml

```diff
@@ -7,23 +7,23 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
   - repo: https://github.com/myint/autoflake
-    rev: v1.4
+    rev: v2.0.0
```

Comment:
Have you updated it with `pre-commit autoupgrade`?

Reply:
I manually update the version. v2.0.0 is used by HCE